src/builder/commands/copy.rs

```diff
@@ -85,6 +91,7 @@ impl Copy {
         .member("include_regex", V::Scalar::new().optional())
         .member("owner_uid", V::Numeric::new().min(0).optional())
         .member("owner_gid", V::Numeric::new().min(0).optional())
+        .member("umask", V::Numeric::new().min(0).max(0o777).optional())
```

Comment:
The default of `umask` as None is unclear. Shouldn't we have default umask of `002` and a `preserve_permissions` flag instead? Or is it for backwards compatibility?